src/Microsoft.TestPlatform.CrossPlatEngine/DataCollection/ProxyDataCollectionManager.cs

```diff
@@ -130,6 +133,13 @@ public void Initialize()
         {
             var port = this.dataCollectionRequestSender.InitializeCommunication();
 
+            // Warn the user that execution will wait for debugger attach.
+            var dataCollectorDebugEnabled = Environment.GetEnvironmentVariable("VSTEST_DATACOLLECTOR_DEBUG");
```

Comment:
Can we move this line after 143, we could print the name and PID of data collector in that case.